### ISSUE TITLE
 get_box_scores() doesnt find any tables since yesterday

### ISSUE THREAD

**Issue by my7hai** [vishaalagartha/basketball_reference_scraper#99]
raw_df = pd.read_html(str(table))[0]
..
.
.
tables = self._parse_tables(self._build_doc(), self.match, self.attrs)

ValueError: No tables found

**Comment by JJnotJimmyJohn** [comment]
Yeah I have to rewrite get_box_scores to scrape from pages like https://www.basketball-reference.com/boxscores/202301010DEN.html.

**Comment by my7hai** [comment]
> Yeah I have to rewrite get_box_scores to scrape from pages like https://www.basketball-reference.com/boxscores/202301010DEN.html.

will you? ive grown very used to this awesome tool =)

**Comment by christian-dalton** [comment]
When will this be fixed?

**Comment by JJnotJimmyJohn** [comment]
> When will this be fixed?

I thought it would be fixed by someone else by now. Kind of busy this week so I'll try fixing it next week if not fixed by then.

**Comment by JJnotJimmyJohn** [comment]
Hey friends! I just submitted a pull request to fix this error. I also developed "ADVANCED" stats mode.
